lib/Test/Nginx/Socket.pm

```diff
@@ -2372,6 +2396,14 @@ to run the specified number of duplicate requests for each test block. When it i
 
 Default to 1.
 
+=head2 stop_after_request
+
+By default, the nginx is not still running after the request. The error log is incompletely, missing logs after nginx stoped.
```

Comment:
@agentzh take a look at this flowchart, we want to get the full error log before calling `check_error_log`.

![qq20170106-0](https://cloud.githubusercontent.com/assets/6814606/21707158/d61708e6-d407-11e6-82ee-8b7aa6e191e4.png)